dotCMS/src/integration-test/java/com/dotcms/graphql/business/GraphqlAPITest.java

```diff
@@ -626,6 +631,76 @@ public void testGetSchema_BothSidedRelationshipFields(final RELATIONSHIP_CARDINA
 
     }
 
+    @DataProvider
+    public static List<Object> dataProviderEEContentTypes() throws Exception {
+        // data provider needs stuff to get initialized because of API access
+        IntegrationTestInitService.getInstance().init();
+
+        // filter only Enterprise content types
+        final List<ContentType> eeTypes = APILocator
+                .getContentTypeAPI(APILocator.systemUser()).findAll().stream()
+                .filter((type)->type instanceof EnterpriseType).collect(Collectors.toList());
+
+        // returns a List of Tuple (typeName, baseType)
+        return eeTypes.stream().map((type)->
+                new Tuple2<>("my"+type.variable(), type.baseType())
+        ).collect(Collectors.toList());
+    }
+
+    @Test
+    @UseDataProvider("dataProviderEEContentTypes")
+    public void testGetSchema_GivenNoEELicense_EnterpriseTypesShouldNotBeAvailableInSchema(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4740432)